lib/dal/src/component.rs

```diff
@@ -2150,16 +2150,29 @@ pub struct ComponentUpdatedPayload {
 #[derive(Clone, Deserialize, Serialize, Debug, PartialEq, Eq)]
 #[serde(rename_all = "camelCase")]
 pub struct ComponentSetPositionPayload {
-    change_set_id: ChangeSetId,
-    component_id: ComponentId,
-    x: i32,
-    y: i32,
-    width: Option<i32>,
-    height: Option<i32>,
-    user_pk: Option<UserPk>,
+    pub change_set_id: ChangeSetId,
+    pub component_id: ComponentId,
+    pub x: i32,
+    pub y: i32,
+    pub width: Option<i32>,
+    pub height: Option<i32>,
+    pub user_pk: Option<UserPk>,
```

Comment:
Do these really need to be public? I'm not catching the use case in this PR

Reply:
Mmm, excellent call out. The previous construction needed it. When I take it away, I get:
```
error[E0616]: field `change_set_id` of struct `ComponentSetPositionPayload` is private
   --> lib/sdf-server/src/server/service/ws/workspace_updates.rs:235:116
    |
235 | ...                   let event = WsEvent::reflect_component_position(self.workspace_pk, payload.change_set_id, payload).await?;
    |                                                                                                  ^^^^^^^^^^^^^ private field
```
Should I make _just_ that one `pub`? Probably better to make an accessor `change_set_id()` right?

Reply:
I think they were needed in the past, but not now. Either way, it's probably no big deal since the mutability of the payload struct is "meh": you either send it or receive it and it's not long lived. Good catch though. I think you're right.

Reply:
@jobelenus Yeah that sounds about right! Getters only  for the things that need to be public. We've been naming then just the name of the field,  so change_set_id() would be the pattern